### ISSUE TITLE
MultiModalClassificationModel issue

### ISSUE THREAD

**Issue by caprone** [ThilinaRajapakse/simpletransformers#792]
training MultiModalClassificationModel  (in my case with simple bert model), two issue arises:

1 --config dict passed to MMBTModel class from self.transformer_config in MultiModalClassificationModel class
doesn't have "use_return_dict" key,  then when test for it issue arises;
(I set manually return_dict to False  in MMBTModel's forward method to walk around..(None is default value).)

2 --  loss computing when train with fp16:
 in row 482 of multi_modal_classification_model.py;
 loss is computed as raw last layer outpt, no cross-entropy  applied;
simply replace with "logits = outputs[0]" and add: "loss = self.criterion(logits, labels)" as for train without fp16;
